kanopi/starter-8
# Add common modules

## Description
> As a developer, I want not to have to re-install the same modules on every site ever.

I analyzed a handful of Kanopi D8 sites using [this query](https://github.com/search?p=3&q=org%3Akanopi+filename%3Acore.extension.yml&type=Code) and came up with a [list of the most frequently used modules](https://docs.google.com/spreadsheets/d/1C45E-k4y3KDs6xq8UloBvuiAcqzdRhRCbIVkM7FhrIM/edit#gid=368973653). Of those, I took the top bunch that's used almost universally and added them to `composer.json` and `init-site` so they're installed from the very start.

## Acceptance Criteria


## Affected URL
NA

## Related Tickets
[Create a sheet of common modules used in last few D8 projects](https://kanopi.teamwork.com/#tasks/22533506)


## Steps to Validate
1. Look at the changed files
1. Run `composer install` to make sure everything installs correctly.


## Deploy Notes
NA
